### DISCUSSION TITLE
[Feature]: Keep Year/Month/Day at top of timeline when scrolling (Mobile+Web)

### DISCUSSION THREAD

**Discussion by meichthys** [immich-app/immich#1667] [Feature Request]
### Feature detail

In order to easier identify the date while scrubbing through photos, it would be nice to have the current photo's Year/Month/Day displayed at the top. Once the user scrolls into the next Year/Month/Day, then the date at the top updates to match that of the current photos being displayed in the timeline.

Example:
![image](https://user-images.githubusercontent.com/10717998/189155962-6ce38599-1973-4a28-8c30-b39d5de72aba.png)

### Platform

Mobile App

**Comment by merRen22** [comment]
I would love to work on this issue for the mobile platform. Can you please assign this to me?

**Comment by bo0tzz** [comment]
@merRen22 I've assigned this issue to you, thanks for offering to work on it!

**Comment by merRen22** [comment]
@bo0tzz I am sorry I would not be able to fix this issue. This goes above my current knowledge I may give it a try in the future.

**Comment by samholton** [comment]
for web: https://github.com/immich-app/immich/pull/7824

**Comment by didekoning** [comment]
It seems like this is removed in a later version. @bo0tzz I would like to see if I can fix this in the new version as a first issue :)

**Reply by didekoning** [reply]
For web: https://github.com/immich-app/immich/pull/25887
